CMSIS/device/stm32f0xx.h

```diff
@@ -1151,7 +1151,7 @@ typedef struct
 #define  DMA_IFCR_CTEIF5                     ((uint32_t)0x00080000)        /*!< Channel 5 Transfer Error clear      */
 
 /*******************  Bit definition for DMA_CCR register  ********************/
-#define  DMA_CCR_EN                          ((uint32_t)0x00000001)        /*!< Channel enable                      */
+#define  DMA_CCR_EN                          ((uint32_t)0x00000001)        /*!<  enable                      */
```

Comment:
This is probably leftover from your last issue. As with the last one, you'll have to undo this.